scripts/common/rook-upgrade.sh

```diff
@@ -141,6 +141,10 @@ function rook_upgrade() {
     if rook_upgrade_is_version_included "$from_version" "$to_version" "1.4" ; then
         addon_source "rookupgrade" "10to14"
         rookupgrade_10to14_upgrade "$from_version"
+
+        # delete both the compressed and decompressed addon files to free up space
+        rm "$DIR/assets/rookupgrade-10to14.tar.gz"
```

Comment:
i dont think this will work for airgap since we use multi-package bundles, you should probably add a -f...

Reply:
Don't the multi-package bundles still decompress to these bundles?

Reply:
No they decompress directly to the file tree, not another archive

Reply:
ouch, ok